cime_config/testdefs/testlist_clm.xml

```diff
@@ -17,7 +17,7 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
-  <test name="SMS_D_Ln1" grid="f09_g17" compset="I1850Clm50BgcNoAnthro" testmods="clm/Meier2022_surf_rough">
+  <test name="SMS_D_Ln6" grid="f09_g17" compset="I1850Clm50BgcNoAnthro" testmods="clm/Meier2022_surf_rough">
```

Comment:
I should have kept this as a separate commit...

This test was failing on SUBMIT with this error:
`Runtime (1800.0 s) must be a multiple of the longest coupling interval ROF_NCPL (10800.0s).  Adjust runtime or ROF_NCPL`
...so I changed the length accordingly.